third_party/blink/renderer/modules/mediasource/source_buffer.h

```diff
@@ -113,6 +113,12 @@ class SourceBuffer final : public EventTargetWithInlineData,
   AudioTrackList& audioTracks();
   VideoTrackList& videoTracks();
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  // Cobalt-specific method that returns the highest presentation
+  // timestamp written to the SbPlayer.
+  double writeHead(ExceptionState& exception_state) const;
```

Comment:
`getWriteHead()`?  Since it's no longer a method mapped from DOM.  Please feel free to keep it as is if this is a convention in Chromium.